olive/passes/onnx/perf_tuning.py

```diff
@@ -342,6 +341,14 @@ def _default_config(accelerator_spec: AcceleratorSpec) -> Dict[str, PassConfigPa
             ),
         }
 
+    @staticmethod
+    def _data_configs() -> Dict[str, PassDataConfigParam]:
+        return {
+            "data_config": PassDataConfigParam(
+                description="Data config for calibration, required if 'dataloader_func' is not provided."
```

Comment:
This is not for `calibration`, right? It is for latency metrics.

Reply:
Oh yeah. I thought I updated the description here but it didn’t 
Thanks!